ui/app/components/auth/form-template.hbs

```diff
@@ -0,0 +1,15 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+{{#if this.formComponent}}
+  {{#let (component this.formComponent) as |AuthComponent|}}
```

Comment:
dynamic component so we instantiate a specific auth type component when one is selected. all of this yielding is thanks to `tabs` which have to render in the parent because selecting a new tab renders different fields. 

alternatively, namespace, options, footer could render here, but i thought it was cleaner to just have all of that in one place